trikot-viewmodels-declarative/sample/android/build.gradle.kts

```diff
@@ -53,6 +53,12 @@ android {
             resources.srcDir("../common/src/commonMain/resources/")
         }
     }
+
+    lint {
+        abortOnError = true
+        checkReleaseBuilds = true
+        disable.add("NotificationPermission")
```

Comment:
there is no way to have this on a more granular level?

Reply:
Possibly, but it's the sample project(s), so I'd say it's good enough.
For more context, this is a weird linting issue triggered by Picasso following the update of the Gradle plugin. Updating Picasso version _may_ have helped, but it seemed less risky to simply disable the check.